test/api_integration/apis/search/bsearch.ts

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import request from 'superagent';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { painlessErrReq } from './painless_err_req';
+import { verifyErrorResponse } from './verify_error';
+
+function parseBfetchResponse(resp: request.Response): Array<Record<string, any>> {
+  return resp.text
+    .trim()
+    .split('\n')
+    .map((item) => JSON.parse(item));
+}
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  describe('bsearch', () => {
+    describe('post', () => {
+      it('should return 200 a single response', async () => {
```

Comment:
All post description should exist in CONST variables.

Reply:
:1st_place_medal: